modules/switchMania/switchMania.ino

```diff
@@ -0,0 +1,58 @@
+/****************************************
+ * Switch Mania
+ *
+ * Léo Paolo
+ * Nicolas-Emmanuel Robert
+ *
+ * Find the right pattern to
+ * defuse the module and prevent
+ * the explosion of the bomb.
+ ***************************************/
+
+#include <ktne_core.h>
+
+#define ON (HIGH)
+#define OFF (LOW)
+
+#define SWITCHES_OF_NUMBER (4)
+
+#define BUTTON (2)
+#define FIRST_SWITCHE (3)
+#define SECOND_SWITCHE (6)
+#define THIRD_SWITCHE (12)
+#define FOURTH_SWITCHE (13)
+
+bool patterns[MAX_DIFFICULTY][SWITCHES_OF_NUMBER] = {{ON, OFF, ON, ON},
+                                                     {OFF, ON, ON, OFF},
+                                                     {OFF, OFF, ON, ON}};
+
+void  patternValidation() {
+  if (digitalRead(FIRST_SWITCHE) == patterns[difficulty - 1][0] &&
+      digitalRead(SECOND_SWITCHE) == patterns[difficulty - 1][1] &&
+      digitalRead(THIRD_SWITCHE) == patterns[difficulty - 1][2] &&
+      digitalRead(FOURTH_SWITCHE) == patterns[difficulty - 1][3]) {
+        defuseModule();
+      }
+   else {
+    applyPenality(20);
```

Comment:
Penality is the same on each difficulty level ?
Maybe we could `applyPenality(20 + (difficulty - 1) * 5)` or anything like that.